src/EnergyPlus/LowTempRadiantSystem.cc

```diff
@@ -1637,6 +1583,52 @@ namespace LowTempRadiantSystem {
 
     }
 
+    void RadiantSystemBaseData::errorCheckZonesAndConstructions(bool &errorsFound,
+                                                                std::string const& cAlphaField3,
+                                                                std::string const& alpha3,
+                                                                std::string const& currentModuleObject,
+                                                                std::string const& alpha1)
+    {
+        Real64 zoneMultipliers = 0.0;
+        Real64 zoneMultipliersSurface = 0.0;
+        Real64 zoneMultiplersTolerance = 0.001;
+        for (int SurfNum = 1; SurfNum <= this->NumOfSurfaces; ++SurfNum) {
+            
+            if (this->SurfacePtr(SurfNum) == 0) continue; // invalid surface -- detected earlier
+            
+            // check zone numbers--ok if they are not the same
+            if (Surface(this->SurfacePtr(SurfNum)).Zone != this->ZonePtr) {
+                ShowWarningError("Surface referenced in " + currentModuleObject +
+                                " not in same zone as Radiant System, surface=" + Surface(this->SurfacePtr(SurfNum)).Name);
+                ShowContinueError("Surface in Zone=" + DataHeatBalance::Zone(Surface(this->SurfacePtr(SurfNum)).Zone).Name +
+                                  " Radiant System in " + cAlphaField3 + " = " + alpha3);
+                ShowContinueError("Occurs in " + currentModuleObject + " = " + alpha1);
+                ShowContinueError("If this is intentionally a radiant system with surfaces in more than one thermal zone,");
+                ShowContinueError("then ignore this warning message.  Otherwise, check the surfaces in this radiant system.");
+            }
+            
+            // check zone multipliers--these must be the same
+            if (SurfNum == 1) zoneMultipliers = double(DataHeatBalance::Zone(this->ZonePtr).Multiplier * DataHeatBalance::Zone(this->ZonePtr).ListMultiplier);
+            zoneMultipliersSurface = double(DataHeatBalance::Zone(Surface(this->SurfacePtr(SurfNum)).Zone).Multiplier
+                                            * DataHeatBalance::Zone(Surface(this->SurfacePtr(SurfNum)).Zone).ListMultiplier);
+            if (std::abs(zoneMultipliers-zoneMultipliersSurface)>zoneMultiplersTolerance) {
+                ShowSevereError("The zone multipliers in " + currentModuleObject + " are not the same for the zones in all of the surfaces");
```

Comment:
How about "The zone multipliers are not the same for all surfaces contained in this radiant system. . . . .